pros/ga/analytics.py

```diff
@@ -68,5 +74,25 @@ def set_use(self, value: bool):
         self.useAnalytics = value
         self.cli_config.ga['enabled'] = self.useAnalytics
         self.cli_config.save()
+    
+    def process_requests(self):
+        responses = []
+        for future in as_completed(self.pendingRequests):
+            try:
+                response = future.result()
+                
+                if not response.status_code==200:
+                    print("Something went wrong while sending analytics!")
```

Comment:
Out of scope for this pr but possibly might look into reading status codes for more specific error messages